src/core/components/name.js

```diff
@@ -0,0 +1,133 @@
+'use strict'
+
+const promisify = require('promisify-es6')
+const series = require('async/series')
+const human = require('human-to-milliseconds')
+const path = require('../ipns/path')
+
+const OFFLINE_ERROR = require('../utils').OFFLINE_ERROR
```

Comment:
const { OFFLINE_ERROR } = require('../utils')

Reply:
It is really strange to me that the errors are in a `utils.js` file, making those imports really strange. In the future, all errors should be easily accessible from all the project, and we should access them in the same way as @fsdiogo suggested.